system/defines.php

```diff
@@ -7,7 +7,8 @@
 
 // Directories and Paths
 if (!defined('GRAV_ROOT')) {
-    define('GRAV_ROOT', getcwd());
+		// Normalize root path (i.e. on Windows convert backslashes to slashes)
+    define('GRAV_ROOT', str_replace(DIRECTORY_SEPARATOR, DS, getcwd()));
```

Comment:
@rhukster I am not gonna touch this for media branch. Should I create a separate issue for it?